I wish_If only/maryna_boieva_1.md

```diff
@@ -0,0 +1,22 @@
+# Exercise 1. Please translate the sentences into English
+
+1. Жаль что вы не пришли пораньше, чтобы оценить всю ущербность архитектуры предложенной студентом первокурсником.
+>If only you had come earlier to check out the lameness of the architecture suggested by a first year student.
+2. Обидно, что мы не должались его прихода, иначе бы мы были в курсе на каком этапе сейчас находится проект и что нам делать с этим дальше.
+>I wish we had waited until his arrival, so we were in touch with the current project stage and knew what should we do next. 
+3. Жаль что мы так и не встретились перед отъездом и не обсудили причину твоего увольнения, у меня много мыслей на этот счет и я хотел бы поделиться ими с тобой.
+>I wish we had met before leaving and discuss the reason of your firing, I have so many thoughts about this I would like to share with you. 
+4. Жаль что проект завершился так и не начавшись ввиду нехватки кадров и соответствующего финансирования. Если бы все сложилось то задумка вполне могла выстрелить на рынке и стать новым хитом.
+>If only the project hadn't ended up before the beginning due the lack of personnel and funding. If the things had been different, the idea could have created a sensation on the market. 
+5. Он пожалел, что пришел и привел с собой друга. На его месте я бы так не поступал. Если бы он пришел вовремя, этого могло и не случиться.
+>He wished, he hadn't come and bring his friend. If I were him, I wouldn't do so. If he had come on time, It might not have happened.
+6. Если бы не ты я не знаю что бы я тогда ответил заказчику, ты повел разговор совсем в другом направлении и нам удалось избежать серьезного разговора о горящих сроках.
+>If not for you, I don't know, what would I have answered to the customer, but you had lead the conversation to the other topic so we could have escaped the serious talk about the deadline.
```

Comment:
`But for you`
but you led the conversation in a different direction
so we avoided a serious conversation